### PR TITLE
Preparing for v0.8.0 release

### LINKED ISSUE #508: [BUG] helm CRD's
**What happened**:

CRD's are not disableable.

**What you expected to happen**:

The CRD's should be in the crds/ directory of the chart as per https://helm.sh/docs/chart_best_practices/custom_resource_definitions/ section: "Method 1: Let helm Do It For You"

I prefer to disable helm from ever managing a CRD because its been quite buggy in the past so I need a way to disable helm from ever touching them to prevent breakage in production

I also would need a way to fetch the crds to apply with kubectl apply -f <someurl> manually outside of the chart.

**How to reproduce it (as minimally and precisely as possible)**:

helm install / upgrade always touches the CRDS. No way to disable that.

**Anything else we need to know?**:

**Environment**:
- Kruise version: 0.7.0

### PR DESCRIPTION
**Pull request by FillZpp**
Signed-off-by: FillZpp <FillZpp.pub@gmail.com>

### Ⅰ. Describe what this PR does
Preparing for v0.8.0 release:

1. prepare chart for v0.8.0
2. add openapi generation
3. update feature-gate and docs

### Ⅱ. Does this pull request fix one issue?

fixes #508 #534

### CODE REVIEW

File: apis/apps/v1alpha1/defaults.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func setDefaultSidecarContainer(sidecarContainer *SidecarContainer) {
 
 func setSidecarSetUpdateStratety(strategy *SidecarSetUpdateStrategy) {
 	if strategy.Type == "" {
-		strategy.Type = NotUpdateSidecarSetStrategyType
+		strategy.Type = RollingUpdateSidecarSetStrategyType
```
Comment (furykerry) [inline apis/apps/v1alpha1/defaults.go:58]: the change in default values should be highlighted in release notes

File: README-zh_CN.md [hunk 2]
```diff
@@ -12,33 +12,32 @@
...
-|Aug 19th, 2020. Kruise v0.6.0 发布! 升级新的项目结构和依赖，并提供新的 Advanced DaemonSet 控制器，详情参见 [CHANGELOG](CHANGELOG.md).|
 
 ## 介绍
 
 OpenKruise (官网: [https://openkruise.io](https://openkruise.io)) 是托管在 [Cloud Native Computing Foundation](https://cncf.io/) (CNCF) 下的 Sandbox 项目。
-它提供一套在 [Kubernetes核心控制器](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/) 之外的扩展 workload 管理和实现。
+它提供一套在 [Kubernetes核心控制器](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/) 之外的扩展工作负载、应用管理能力。
 
-目前，Kruise 提供了以下 workload 控制器：
+目前，Kruise 主要提供了以下 CRD：
```
Comment (furykerry) [inline README-zh_CN.md:24]: the previous description sounds better, just add items for ImagePullJob is enough, NodeImage is just an implementation detail

File: apis/apps/v1alpha1/defaults.go (hunk 1 above)
Comment (furykerry) [reply apis/apps/v1alpha1/defaults.go:58]: it seems that the default value is RollingUpdateSidecarSetStrategyType in previous release

File: charts/kruise/v0.8.0/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+| `daemon.resources.limits.memory`          | Memory resource limit of kruise-daemon container             | `64Mi`                        |
+| `daemon.resources.requests.cpu`           | CPU resource request of kruise-daemon container              | `0`                           |
+| `daemon.resources.requests.memory`        | Memory resource request of kruise-daemon container           | `0`                           |
+| `daemon.affinity`                         | Affinity policy for kruise-daemon pod                        | `{}`                          |
+| `webhookConfiguration.failurePolicy.pods` | The failurePolicy for pods in mutating webhook configuration | `Ignore`                      |
+| `webhookConfiguration.timeoutSeconds`     | The timeoutSeconds for all webhook configuration             | `30`                          |
+| `crds.managed`                            | Kruise will not install CRDs with chart if this is false     | `true`                        |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
```
Comment (furykerry) [inline charts/kruise/v0.8.0/README.md:66]: add Optional feature-gate description?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,91 @@
...
+2. SidecarSet has been refactored. Make sure there is no SidecarSet being upgrading when you upgrade Kruise,
+   and read [the latest doc for SidecarSet](https://openkruise.io/en-us/docs/sidecarset.html).
+3. A new component named `kruise-daemon` comes in. It is deployed in kruise-system using DaemonSet, defaults on every Node.
+
+Now Kruise includes two components:
+
+- **kruise-controller-manager**: contains multiple controllers and webhooks, deployed using Deployment.
+- **kruise-daemon**: contains bypass features like image pre-download and container restart in the future, deployed using DaemonSet.
+
+### New CRDs: NodeImage and ImagePullJob
```
Comment (furykerry) [inline CHANGELOG.md:17]: please add description and links in docs/README.md

**Review by furykerry — CHANGES_REQUESTED** [review]

**Review by Fei-Guo — APPROVED** [review]

**Review by furykerry — APPROVED** [review]

**Pull request merged**